bib/cmd/bootc-image-builder/main.go

```diff
@@ -43,6 +43,29 @@ type BuildConfig struct {
 	Blueprint *blueprint.Blueprint `json:"blueprint,omitempty"`
 }
 
+// canChownInPath checks if the ownership of files can be set in a given path.
+func canChownInPath(path string) (bool, error) {
+	if info, err := os.Stat(path); err == nil {
```

Comment:
I wonder if we could use the standard go convention here, i.e.
```go
	info, err := os.Stat(path)
	if err != nil {
		return false, err
	}
	if !info.IsDir() {
		return false, fmt.Errorf("%s is not a directory", path)
	}
```
or am I missing something?

Reply:
Not missing anything.  I think I was just having a weird brain day.